### PR TITLE
MAP-1457 111 Cover PrisonSignedOperationCapacity toString


### PR DESCRIPTION
**Pull request by marcusaleman**
Tiny PR to cover some red lines in KOVER report.
toString method.
image

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt [hunk 1]
```diff
@@ -16,6 +16,10 @@ import java.time.LocalDateTime
 @Transactional
 class PrisonSignedOperationalCapacityRepositoryTest : TestBase() {
 
+  val mdi = "MDI"
```
Comment (Mjwillis) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt:19]: Not sure why this is needed

File: src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt [hunk 2]
```diff
@@ -16,6 +16,10 @@ import java.time.LocalDateTime
...
+  val xxx = "XXX"
```
Comment (Mjwillis) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt:20]: name of a variable called `xxx` is unclear

File: src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt (hunk 1 above)
Comment (Mjwillis) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt:19]: You should name this something like `testPrisonId`

**Review by Mjwillis — COMMENTED** [review]
Minor comments

File: src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt (hunk 2 above)
Comment (marcusaleman) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/PrisonSignedOperationalCapacityRepositoryTest.kt:20]: testPrisonId **OK**, xxx was typo/left behind. corrected thanks

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**